### DISCUSSION TITLE
Bounce rate over time

### DISCUSSION THREAD

**Discussion by ghost** [plausible/analytics#1093] [Anything Else]
Plausible shows a graph of visitors over time. Could it also show bounce rate over time?

**Comment by metmarkosaric** [comment]
we plan to work on this soon after releasing the custom filter. custom filter will hopefully be ready next week. you can see more on our short term plans in our roadmap https://github.com/plausible/analytics/projects/1

**Comment by ghost** [comment]
Is it now possible to see bounce rate over time?

**Reply by metmarkosaric** [reply]
not yet but should go live pretty soon. it's currently on our staging for testing and we need to fix couple of things before making it live to all. it's the first thing on our list currently so do stay tuned!

**Reply by metmarkosaric** [reply]
this is now live!
